src/subclass/mod.rs

```diff
@@ -2,22 +2,18 @@
 // See the COPYRIGHT file at the top-level directory of this distribution.
 // Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
 
-pub mod application;
+//pub mod application;
 pub mod application_window;
-pub mod bin;
 pub mod box_;
-pub mod container;
 pub mod dialog;
 pub mod header_bar;
 pub mod widget;
 pub mod window;
 
 pub mod prelude {
-    pub use super::application::GtkApplicationImpl;
+    //pub use super::application::GtkApplicationImpl;
```

Comment:
Why is this commented out?

Reply:
Because I didn't had gio.Application in manual and it didn't compile, I will get it back.

Reply:
Ah, okay :)